src/main/pages/che-7/end-user-guide/proc_defining-maven-settings-xml-file-across-workspaces.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// using-maven-artifact-repositories
+
+[id="proc_defining-maven-settings-xml-file-across-workspaces_{context}"]
+= Defining maven `settings.xml` file across workspaces
+
+To specify your own `settings.xml` file across workspaces, use the Secret with `settings.xml` and define labels and annotation to mount it as a file (see link:{site-baseurl}che-7/mounting-a-secret-as-a-file-or-an-environment-variable-into-a-workspace-container/#mounting-a-secret-as-a-file-into-a-workspace-container_mounting-a-secret-as-a-file-or-an-environment-variable-into-a-workspace-container[Mounting a secret as a file into a workspace container]).
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: maven-settings-secret
+  labels:
+    app.kubernetes.io/part-of: che.eclipse.org
+    app.kubernetes.io/component: workspace-secret
+  annotations:
+    che.eclipse.org/target-container: maven
+    che.eclipse.org/mount-path: /home/user/.m2
+    che.eclipse.org/mount-as: file
+type: Opaque
+stringData:
+  settings.xml: |-
+    <settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0
+                          https://maven.apache.org/xsd/settings-1.0.0.xsd">
+      ...
+    </settings>
+----
+
+In this case, you can't set whole `<home dir>/.m2` as a volume, because it will clash with Secret's `mount-path`. If you still want to persist local maven repository, use `containerPath: <home dir>/.m2/repository` or set different local repository path in your `settings.xml` and define volume in your devfile for this path.
```

Comment:
```suggestion
In this case, you cannot set the whole `__<home-dir>__/.m2` as a volume because it would clash with the `mount-path`. To persist a local Maven repository, use `containerPath: __<home-dir>__/.m2/repository`, or set a different local repository path in your `settings.xml` and define a volume in your devfile for this path.
```